### PR TITLE
FRON-13 Implement landing page

### PR DESCRIPTION
**Pull request by MaliniakovaKarin**

### CODE REVIEW

File: src/components/Navigation.vue [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+
+<script>
+
+export default {
+	data: function() {
+		return {
+		};
+	}
+};
+</script>
```
Comment (OliverKanat) [inline src/components/Navigation.vue]: Add newline

**Review by OliverKanat — DISMISSED** [review]

File: src/components/Navigation.vue (hunk 1 above)
Comment (MCFreddie777) [reply src/components/Navigation.vue]: Reason?

**Comment by MCFreddie777** [comment]
I'll store your images using GIT LFS.

**Review by MCFreddie777 — APPROVED** [review]
Looks good to me. Saw some nitpicks like bad formatting or unnecessary newlines, but we're format it altogether later. 

Keep the good work & feel free to merge! 👍

**Pull request merged**